lib/data-plane/dataplane-ssh-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+data:
+  authorized_keys: _replaced_
+  ssh-privatekey: _replaced_
+  ssh-publickey: _replaced_
+kind: Secret
+metadata:
+  name: dataplane-ansible-ssh-private-key-secret
```

Comment:
I'm finding the mixed directory/file name of `data-plane` and `dataplane` super awkward...

Reply:
Yes, I think we should name it dataplane since that's what the [dataplane-operator](https://openstack-k8s-operators.github.io/dataplane-operator) is already using.

Reply:
since we're introducing the `data-plane` with this change, can we change it to `dataplane`?

Reply:
Yes, let's use `dataplane` everywhere here

Reply:
Done